dark-mode.css

```
@@ -170,7 +170,7 @@
     background-color: hsla(0, 0, 100%, 0.1);
   }
 
-  body.dark-mode .rc-modal {
+  body.dark-mode .rc-modal .rc-modal__footer {
```

I think it should be this, or else the `.rc-modal` doesn't get the style applied too:
```suggestion
  body.dark-mode .rc-modal,
  body.dark-mode .rc-modal__footer {
```